tesseract_process_managers/include/tesseract_process_managers/task_generators/fix_state_bounds_task_generator.h

```diff
@@ -51,6 +51,12 @@ struct FixStateBoundsProfile
 
   /** @brief Maximum amount the process is allowed to correct. If deviation is further than this, it will fail */
   double max_deviation_global = std::numeric_limits<double>::max();
+
+  /** @brief Amount to reduce the upper bounds before clamping limits. Should be > 1 */
+  double upper_bounds_reduction{ 1e-6 };
```

Comment:
Should we use float epsilon here as the default?

Reply:
Yeah that would probably work

Reply:
I rebased and changed this